diff --git a/dist/routes/rest/index.d.ts b/dist/routes/rest/index.d.ts
index 03d72746cf93c23ded5f91b52ae527ef850727d3..c56b285bc1c1498568cdc825102b6985e32c56fc 100644
--- a/dist/routes/rest/index.d.ts
+++ b/dist/routes/rest/index.d.ts
@@ -1,32 +1,32 @@
 import { type SanitizedConfig } from 'payload';
 export declare const OPTIONS: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 export declare const GET: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 export declare const POST: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 export declare const DELETE: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 export declare const PATCH: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 export declare const PUT: (config: Promise<SanitizedConfig> | SanitizedConfig) => (request: Request, args: {
     params: Promise<{
-        slug: string[];
+        slug?: string[];
     }>;
 }) => Promise<Response>;
 //# sourceMappingURL=index.d.ts.map
