diff --git a/dist/fields/MetaImage/MetaImageComponent.js b/dist/fields/MetaImage/MetaImageComponent.js
index 1eff9acd769613155c720afb32a97657b8685d38..942db871e213e6de1a819b4cfc2fc9a727420d44 100644
--- a/dist/fields/MetaImage/MetaImageComponent.js
+++ b/dist/fields/MetaImage/MetaImageComponent.js
@@ -1,8 +1,8 @@
 'use client';
 import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { FieldLabel, RenderCustomComponent, UploadInput, useConfig, useDocumentInfo, useField, useForm, useLocale, useTranslation } from '@payloadcms/ui';
 import { reduceToSerializableFields } from '@payloadcms/ui/shared';
 import React, { useCallback } from 'react';
 import { Pill } from '../../ui/Pill.js';
 export const MetaImageComponent = (props)=>{
     const { field: { label, localized, relationTo, required }, hasGenerateImageFn, readOnly } = props;
@@ -37,7 +37,7 @@ export const MetaImageComponent = (props)=>{
             },
             method: 'POST'
         });
-        const generatedImage = await genImageResponse.text();
+        const { result: generatedImage } = await genImageResponse.json();
         setValue(generatedImage || '');
     }, [
         hasGenerateImageFn,
